spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../person'
+
+describe Person do
+  before :each do
+    @person = Person.new(0o1, 22, 'Arthur')
+  end
+
+  describe 'name_of_peson' do
+    it 'should be the same name' do
+      expect(@person.name).to eql 'Arthur'
+    end
+  end
+
+  describe 'age_of_person' do
+    it 'should be the same age' do
+      expect(@person.age).to eql 22
+    end
+  end
```

Comment:
- [ ] Add different negative and positive test case scenarios for the **can_use_services?**.

- [ ] Add tests for the **add_rentals** method in the Person class